scrapper.py

```diff
@@ -44,20 +79,77 @@ def get_search_urls(self):
         pass
 
 
+class Article:
+    def __init__(self, article):
+        self.article = article
+
+
+class ArticleParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
```

Comment:
you do not follow the requirements about what should be done during instantiation of the parser